### DISCUSSION TITLE
currentRouteName() and wire:poll issue?

### DISCUSSION THREAD

**Discussion by flatcapco** [livewire/livewire#6645] [1. Help]
I have a blade component for a navigation in my app, when using wire:poll the currentRouteName() is changed to livewire.update. The knock on effect is that I cannot apply an active class to my navigation like so:

class="{{ (Route::currentRouteName() == 'booking') ? $active_class : $class }}"

As soon as poll runs currentRouteName == 'livewire.update'

I really don't want to overly complicate the component if possible and ideally would just leave it as a blade component not a livewire component - is that my best option though?

Thanks!
Alex

**Comment by alexcutts2** [comment]
In these situations I tend to apply wire:ignore to the navigation div and it will not change in future updates of the component data.

**Comment by bucari** [comment]
i have the same issue in a page with filter. The filter fields change base on Route::currentRouteName() but when i set a filter the currentRouteName is livewire.update

I cannot use wire:ignore because i have a searchable select that needs to update.
